app/bibleview-js/src/components/modals/AskBookmarkSettings.vue

```diff
@@ -0,0 +1,805 @@
+        <button
+          type="button"
+          class="tab-button"
+          :class="{ active: activeTab === 'icons' }"
+          @click="activeTab = 'icons'"
+        >
+          <FontAwesomeIcon :icon="faIcons" />
+          {{ strings.customIconLabel }}
+        </button>
+        <button
+          type="button"
+          class="tab-button"
+          :class="{ active: activeTab === 'editAction' }"
+          @click="activeTab = 'editAction'"
+        >
+          <FontAwesomeIcon :icon="faEdit" />
+          {{ strings.editActionLabel }}
+        </button>
+      </div>
+
+      <!-- Tab Content -->
+      <div class="tab-content">
+        <!-- Custom Icon Tab -->
+        <div v-if="activeTab === 'icons'" class="tab-panel">
+          <div class="icon-list">
+            <div
+              v-for="[key, icon] in Array.from(customIconMap.entries())"
+              :key="key"
+              class="icon-item"
+              :class="{selected: key === selectedIcon}"
+              @click="selectIcon(key)">
+              <FontAwesomeIcon :icon="icon" />
+            </div>
+            <div
+               class="icon-item"
+               :class="{selected: selectedIcon === null}"
+               @click="selectIcon(null)">
+               <FontAwesomeIcon :icon="faTimes" />
+            </div>
+          </div>
+        </div>
+
+        <!-- Edit Action Tab -->
+        <div v-if="activeTab === 'editAction'" class="tab-panel">
+          <div class="edit-action-controls">
+            <div class="mode-selection">
+              <label>{{ strings.editActionModeLabel }}:</label>
+              <div class="mode-toggle-buttons">
+                <button
+                  type="button"
+                  class="mode-toggle"
+                  :class="{ active: selectedEditAction.mode === null }"
+                  @click="selectedEditAction.mode = null"
+                  :title="strings.editActionModeNone">
+                  <FontAwesomeIcon :icon="faBan" />
+                  <span>{{ strings.editActionModeNone }}</span>
+                </button>
+                
+                <button
+                  type="button"
+                  class="mode-toggle"
+                  :class="{ active: selectedEditAction.mode === EditActionMode.APPEND }"
+                  @click="selectedEditAction.mode = EditActionMode.APPEND"
+                  :title="strings.editActionModeAppend">
+                  <FontAwesomeIcon :icon="faPlus" />
+                  <span>{{ strings.editActionModeAppend }}</span>
+                </button>
+                
+                <button
+                  type="button"
+                  class="mode-toggle"
+                  :class="{ active: selectedEditAction.mode === EditActionMode.PREPEND }"
+                  @click="selectedEditAction.mode = EditActionMode.PREPEND"
+                  :title="strings.editActionModePrepend">
+                  <FontAwesomeIcon :icon="faArrowUp" />
```

Comment:
append and prepend icons should be changed. There must be better options in fontawesome.